README.md

```diff
@@ -39,33 +43,28 @@ You can find the full reference on [Algolia's website](https://www.algolia.com/d
 
 1. **[Search UI](#search-ui)**
 
-    * [index.html](#indexhtml)
-
-
 
+1. **[List of available methods](#list-of-available-methods)**
 
 
 # Getting Started
 
 
 
-
 ## Supported platforms
 
 Compatibilities:
   * `.NET Framework 4.6`
   * `.NET Framework 4.6.2`
   * `.NET Core 1.0`
   * `.NET Core 1.1`
-  * `.NET Core 2.0`
```

Comment:
I opened a PR to add these version: https://github.com/algolia/doc/pull/1329
The PR will be updated next time we deploy 👌